customize.dist/pages/login.js

```diff
@@ -15,28 +15,36 @@ define([
                     h('div.col-md-3'),
                     h('div#userForm.form-group.hidden.col-md-6', [
                         h('div.cp-login-instance', Msg._getKey('login_instance', [ Pages.Instance.name ])),
-                        h('input.form-control#name', {
-                            name: 'name',
-                            type: 'text',
-                            autocomplete: 'off',
-                            autocorrect: 'off',
-                            autocapitalize: 'off',
-                            spellcheck: false,
-                            placeholder: Msg.login_username,
-                            autofocus: true,
-                        }),
-                        h('input.form-control#password', {
-                            type: 'password',
-                            'name': 'password',
-                            placeholder: Msg.login_password,
-                        }),
+                        h('div.big-container', [
+                            h('div.input-container', [
+                                h('label.label', { for: 'name' }, Msg.login_username + ':'), // XXX label not visible
```

Comment:
here and in all similar cases please keep the comment about non visible labels as I'm still unsure what course of action is best